templates.json

```diff
@@ -518,5 +542,31 @@
                 "successDebugMessage": "Hello Torizon!"
             }
         }
+    ],
+    "DeprecatedTemplates": [
```

Comment:
I would prefer to create another JSON file `deprecated.json` instead to move the deprecated templates to the `DeprecatedTemplates`. Also do not forget to adjust the `projectUpdater.ps1` to get the data from this new file if the current project does not match one of the `templates.json`.

Reply:
Sure, will create a new one